config.json

```diff
@@ -46,6 +46,17 @@
         "Transforming"
       ]
     },
+	{
+      "uuid": "2ee3cc7a-db3f-4668-9983-ed6d0fea95d1",
+      "slug": "two-fer",
+      "core": true,
```

Comment:
I agree this should be a core exercise, but we can probably then remove the `binary-search-tree` as a core exercise as its topics will already have been introduces by earlier exercises. 

We should also have the `two-fer` exercise unlock a couple of non-core exercises, which might require some reshuffling due to the possible removal of the `binary-search-tree` as a core exercise, which will then no longer unlock any exercises.